Dockerfile

```diff
@@ -370,40 +407,53 @@ RUN if [ "${CPAN_PKGS}" != "" ] || [ "${PIP_PKGS}" != "" ] || [ "${IMAGE_LAYER_P
       && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* ~/.[^.] ~/.??* ~/* \
     ; fi
 
+# Custom image layers options:
+ARG IMAGE_LAYER_PYTHON="1"
+ARG IMAGE_LAYER_PYTHON_EXT="1"
+
+# Custom installation packages
+ARG PIP_PKGS=""
+
 # Add Python app layer
 RUN if [ "${PIP_PKGS}" != "" ] || [ "${IMAGE_LAYER_PYTHON}" != "0" ] || [ "${IMAGE_LAYER_PYTHON_EXT}" != "0" ]; then \
       LC_ALL=C DEBIAN_FRONTEND=noninteractive apt-get update \
       && LC_ALL=C DEBIAN_FRONTEND=noninteractive apt-get install -qqy --no-install-recommends \
-          libinline-python-perl \
-          python3 \
-          python3-dev \
-          python3-pip \
-          python3-setuptools \
-          python3-wheel \
```

Comment:
pinning was intentionally left out because there is no real value not to use the latest and greatest version...

Reply:
pinning is implemented, to invalidate the cache if a new version should be uses